src/main/resources/com/google/api/codegen/go/sample.snip

```diff
@@ -14,6 +14,8 @@
     //    If not already done, install the gcloud CLI from
     //    https://cloud.google.com/sdk/ and run
     //    `gcloud beta auth application-default login`
+    //    For more information, see:
```

Comment:
I'm also not much a fan of the hanging indents.
Added the period in the meanwhile, PTAL